test/built-ins/String/prototype/matchAll/invoke-builtin-matchall.js

```diff
@@ -1,43 +0,0 @@
-// Copyright (C) 2018 Jordan Harband. All rights reserved.
```

Comment:
This test was removed as per changes from https://github.com/tc39/proposal-string-matchall/pull/33
  - Given a string, @@matchAll is no longer executed on the internally created
    RegExp.